Edits to PR 5205

cynthn opened the pull request.

azurecla commented:
Hi __@chaosrealm__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (eugenesh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
@chaosrealm 
As a Microsoft employee, you need to start submitting your pull requests through the private repo azure-content-pr. For guidance, see the contributor guide https://github.com/Azure/azure-content/tree/master/contributor-guide and the pull request submission procedure https://github.com/Azure/azure-content/blob/master/contributor-guide/git-commands-for-master.md in that contributor guide.

I'll merge this pull request, but in the future please use the private repo. Thanks.

Commit:
Upload file using azcopy

Specifies whether the destination blob is a block blob, a page blob, or an append blob. This option is applicable only when you are uploading a blob. Otherwise, an error is generated. If the destination is a blob and this option is not specified, by default, AzCopy creates a block blob.

Commit:
removing extra code blocks
